packages/common/src/service.js

```diff
@@ -51,14 +51,13 @@ function Service(config, options) {
   });
 
   this.makeAuthenticatedRequest = util.makeAuthenticatedRequestFactory(reqCfg);
-
   this.authClient = this.makeAuthenticatedRequest.authClient;
   this.baseUrl = config.baseUrl;
   this.getCredentials = this.makeAuthenticatedRequest.getCredentials;
   this.globalInterceptors = arrify(options.interceptors_);
   this.interceptors = [];
   this.packageJson = config.packageJson;
-  this.projectId = options.projectId;
+  this.projectId = options.projectId || '{{projectId}}';
```

Comment:
GrpcService doesn't currently support interceptors. That should work, though.

Task list, if we go this route:

- [ ] Support request interceptors for GrpcService
- [ ] Make all request interceptors compatible with async modifications